<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="search_2">
    <item>
     <layout class="QHBoxLayout" name="search">
      <item>
       <widget class="QLineEdit" name="search_input">
        <property name="placeholderText">
         <string>请输入搜索对象</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="search_type">
        <property name="currentText">
         <string>单曲</string>
        </property>
        <item>
         <property name="text">
          <string>单曲</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>歌单</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>歌手</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_button">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="list"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>暂时还没有歌曲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>《</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>》</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
